locales/en.default.json

```diff
@@ -104,8 +104,12 @@
       "on_sale": "Sale",
       "product_variants": "Product variants",
       "media": {
-        "open_featured_media": "Open featured media in gallery view",
-        "open_media": "Open media {{ index }} in gallery view",
+        "gallery_viewer": "Gallery Viewer",
+        "load_image": "Load image {{ index }} in gallery view",
+        "load_model": "Load 3D Model {{ index }} in gallery view",
+        "load_video": "Play video {{ index }} in gallery view",
+        "image_available": "Image {{ index }} is now available in gallery view",
```

Comment:
Question to @svinkle: should this be announced every time the user clicks the thumbnail or just on the first time the image finishes loading? 

If it's just for the first time, I'm unsure if we can guarantee this to fire, as browsers may skip the `onload` event if the image is cached. For now, it's implemented as a one-time only. Let me know if we should change to always display this message.

Reply:
Each time, as is the case with a video loading (audio playback indicates a successful load) and a 3D model (built-in live region announces "loaded" state.)